src/phase_utils.py

```diff
@@ -205,8 +208,76 @@ def __str__(self):
         return f"lp-{self._id}"
 
 
+class AnnotationStation:
+    """
+    Used when annotating songs in the user phase. Used for saving results
+    from gui to file.
+    Reducing the need for annotating the same song more than once.
+
+    Annotations will be saved as a dictionsary in following format
+    (with annotations for song_ids = 1, 2):
+    {'1': [[arousal1], [valence1]], '2': [[arousal2], [valence2]]}
+    """
+
+    def __init__(self, path: Path):
+        """
+        AnnotationStation constructor
+
+        Args:
+            path (Path): Path to dictionary in json format.
+                    (Note: include .json tag.)
+
+        Raises:
+            FileNotFoundError: Raises error when path is not correct.
```

Comment:
"Raises error" already specified. Skip this i.e. "Raises: FileNotFoundError: When the path is not valid."